lms/djangoapps/teams/tests/test_api.py

```diff
@@ -93,10 +95,12 @@ def setUpClass(cls):
             topic_id=TOPIC2
         )
         cls.team3 = CourseTeamFactory(course_id=COURSE_KEY2, team_id='team3', topic_id=TOPIC3)
+        cls.team4 = CourseTeamFactory(course_id=COURSE_KEY2, team_id='team4', topic_id=TOPIC4)
 
         cls.team1.add_user(cls.user1)
         cls.team1.add_user(cls.user2)
         cls.team2.add_user(cls.user3)
+        cls.team4.add_user(cls.user3)
```

Comment:
same user 3? in the previous teams they use a different user

Reply:
`user3` is part of multiple teams, this user is used to make sure there are no conflicts with other teams